Ports/jakt/package.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=jakt
+version=git
+useconfigure=true
+depends=(llvm)
+commit_hash=063e9767ff80db1a1cfe1a805cc8b7e2e577d9f3
```

Comment:
Why don't we simply use this as a version?

Reply:
:shrug:
```
❯ rg version=git Ports
Ports/stpuzzles/package.sh
4:version=git

Ports/qt6-serenity/package.sh
3:version=git

Ports/SDLPoP/package.sh
4:version=git

Ports/sl/package.sh
3:version=git

Ports/doom/package.sh
4:version=git

Ports/Another-World/package.sh
4:version=git

Ports/nyancat/package.sh
3:version=git

Ports/Super-Mario/package.sh
4:version=git

Ports/chester/package.sh
4:version=git
```